runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -53,10 +53,12 @@
 @RunWith(MockitoJUnitRunner.class)
 public class SlayerPluginTest
 {
+	private static final String TASK_GEM_PROGRESS = "You're assigned to kill Dust Devils; only 317 more to go.";
+	private static final String TASK_GEM_PROGRESS_WILDERNESS = "You're assigned to kill Dust Devils in the Wilderness; only 222 more to go.";
+	private static final String TASK_NEW_FROM_PARTNER = "You have received a new Slayer assignment from breaklulz: Dust Devils (377)";
+
 	private static final String TASK_NEW = "Your new task is to kill 231 Suqahs.";
 	private static final String TASK_NEW_NPC_CONTACT = "Excellent, you're doing great. Your new task is to kill<br>211 Suqahs.";
-	private static final String TASK_CHECKSLAYERGEM_WILDERNESS = "You're assigned to kill Suqahs in the Wilderness; only 211 more to go.";
```

Comment:
Why are these old test strings and tests removed? Just add the one new string and test.

Reply:
They wrote new tests instead of using the old test